clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -44,6 +44,12 @@ from functools import wraps
 import google.protobuf.text_format
 from google.rpc import status_pb2, code_pb2
 
+try:
+    from idpf import p4info_pb2 as idpf_p4info
+    from idpf import p4runtime_pb2 as idpf_p4runtime
+except:
+    print("Unable to import idpf python packages")
```

Comment:
As I mentioned during our conversation, the message should tell the user what effect the missing packages will have, and how to fix the problem.
```
Unable to import the idpf python modules. You will need to install an updated
version of the the "p4runtime" package in order to configure meters.
```
I'm not sure that _configure meters_ is the right expression. Perhaps _in order to use the meter commands_?